packages/app/components/FormLogin/GoToRegister/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useTranslation } from "app/common-i18n/use-translation";
+import { Link } from "solito/link";
+import { StyleSheet, Platform } from "react-native";
+import { ButtonCta } from "../../Buttons";
+import CompositionSection from "../../Compositions/CompositionSection/index.web";
```

Comment:
You work with mobile application, but make import for web. If you need this component for native, create it